_includes/card-extended.html

```diff
@@ -0,0 +1,100 @@
+    {% if icon_content contains '/' or icon_content contains '.svg' %}
+        {% assign icon_svg = true %}
+    {% endif %}
+{% endunless %}
+
+{% assign card_class = 'cards cards-wide' | strip %}
+
+<div class="{{ card_class }}">
+{% if no_icon==false %} 
+        <span class="cards-icon">
+            {% if icon_svg %}
+                {% include {{ icon_content }} %}
+            {% else %}
+                <i class="material-icons{% if icon_outlined %}-outlined{% endif %}">
+                    {{ icon_content }}
+                </i>
+                {% endif %}
+        </span>
+{% endif %}
+    {% if include.card_demoshop %}
+    <div>
+    {% endif %}
+        <span class="cards-content">
+            <span class="{{ title_type }}">{{ include.title }}</span>
+            <span>{{ text | markdownify }}</span>
+            <table class="table table-plain">
+                <thead>
+                    <tr>
+                        {% for header in include.header %}
+                        <th> {% if header.badge_type %} 
+                            <span class="badge badge-{{header.badge_type}}">{{header.table_header}}</span>
+                            {% else %}
+                            {{ header.table_header }}
+                            {% endif %}
+                        </th>
+                        {% endfor %}
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for table_content in include.table_content %}
+                    <tr>
+                        
+                        <th scope="row">
+                            <i class="material-icons{% if icon_outlined %}-outlined{% endif %}">
+                                {{table_content.icon}}
+                            </i> {{table_content.label}}
+                            {% if table_content.swedbankPay %}
+                            <td>
+                                <i class="material-icons{% if icon_outlined %}-outlined{% endif %}">
+                                    check_circle_outline
+                                </i>
+                            </td>
+                            {% endif %}
+                            {% if table_content.merchantSide %}
+                            <td>
+                                <i class="material-icons{% if icon_outlined %}-outlined{% endif %}">
+                                    check_circle_outline
+                                </i>
+                            </td>
+                            {% endif %}
+                        </th>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        </span>
+        {% unless disabled %}
+        <a class="btn btn-{{include.button_type}} {{include.button_alignment}}" href="{{to}}" type="button">{{ include.button_content }}<i class="material-icons ml-2"> arrow_forward</i></a>
+        {% endunless %}
+    {% if include.card_demoshop %}
```

Comment:
I don't think `card_demoshop` is reusable enough to warrant a place in the theme. It would be better to just do this custom in the Developer Portal somehow, I think. Perhaps with `{% contentblock … %}` and `{% contentfor … %}`? See how it's used in `front-page.html` and `page1.md`.